src/test/java/com/thindeck/dynamo/ClasspathRule.java

```diff
@@ -97,6 +97,7 @@ public boolean apply(final Class<?> input) {
                         && !name.endsWith("ITCase")
                         && !name.endsWith("ClasspathRule")
                         && !name.endsWith("RepoRule")
+                        && !name.endsWith("DyTxn")
```

Comment:
@pinaf  Yes, because the checkImmutabilityTest checks that all classes in package com.thindeck.dynamo have @Immutable annotation